src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -1,19 +1,13 @@
 package seedu.addressbook.commands;
 
-import java.util.HashSet;
-import java.util.Set;
-
 import seedu.addressbook.data.exception.IllegalValueException;
-import seedu.addressbook.data.person.Address;
-import seedu.addressbook.data.person.Email;
-import seedu.addressbook.data.person.Name;
-import seedu.addressbook.data.person.Person;
-import seedu.addressbook.data.person.Phone;
-import seedu.addressbook.data.person.ReadOnlyPerson;
-import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.person.*;
```

Comment:
Imported classes should always be listed explicitly. https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements